### ISSUE TITLE
Webpage stuck at loading for chrome

### ISSUE THREAD

**Issue by aldwnesx** [fullcalendar/fullcalendar#5558]
I am facing a really weird issue.
I am using this library and the codes I am using is from the documentation.

```
  selectable: true,

    initialView: 'dayGridMonth',
    headerToolbar: {
      left: 'prev,next today',
      center: 'title',
      right: 'dayGridMonth,timeGridWeek,timeGridDay'
    },
    events: [

    ]
  });

  calendar.render();

```

If I leave `events` empty, on Google Chrome, the website stucks at an infinite loading state until chrome tells me it's unresponsive. I do not see any error messages on console.
However I tried opening the page on other browsers such as mozzila firefox and microsft edge and it's about to load fine.

**Comment by arshaw** [comment]
Could someone please post a repro? (the more the merrier). I can't get it to recreate on my Chrome.

https://fullcalendar.io/reduced-test-cases

A recreation would be valuable to see what version you are using as well.

Also, it'd be great to know your browser window's dimensions when this happens.

**Comment by aldwnesx** [comment]
> Could someone please post a repro? (the more the merrier). I can't get it to recreate on my Chrome.
> 
> https://fullcalendar.io/reduced-test-cases
> 
> A recreation would be valuable to see what version you are using as well.
> 
> Also, it'd be great to know your browser window's dimensions when this happens.

I managed to figure out the cause.

I have the div placed inside bootstrap 4 row column as such:

               
<div class="row">
 <div class="col">
    <div id='calendar'></div>
</div>

</div>


Then I am getting the issue. If I place it outside there is no issue!

**Comment by arshaw** [comment]
Thanks @aldwnesx, it'd be great if you could share a runnable test case of your scenario as well, if possible

**Comment by NicolasCARPi** [comment]
Dear Adam,

I can reproduce this issue, too. Archlinux with Chromium 83.0.4103.116-2.

The page is blank and crashes, nothing printed in the console.

Using boostrap v4 and fullcalendar 5.1.0. The issue appears only if `themeSystem: 'bootstrap'` is set. So that narrows it down a bit. Unlike @aldwnesx the "calendar div" is not in a row. So I'm not sure how @aldwnesx managed to resolve the issue (also his message is missing a part, which doesn't help).

I really tried my best to use the codepen templates to get a reproducible issue, but didn't manage to. Even with all the nice tools we have at our disposal, it's still difficult to get a reproducible state, because jquery, webpack, typescript, css-loader, fontawesome, babel, plugins and many more all play a role. But because the project is open source, at least I can provide the code that makes it crash (on a WIP branch):

[webpack config](https://github.com/elabftw/elabftw/blob/calendarv5/builder.js)
[package.json](https://github.com/elabftw/elabftw/blob/calendarv5/package.json)
[typescript code with likely the culprit hiding in there](https://github.com/elabftw/elabftw/blob/calendarv5/src/ts/team.ts)
[the html part](https://github.com/elabftw/elabftw/blob/calendarv5/src/templates/team.html#L51-L87)

So all I can say is that **this bug exists** ;)

**Comment by NicolasCARPi** [comment]
Little followup: I have finished the migration to v5 and this bug doesn't appear anymore in Chromium...

**Comment by arshaw** [comment]
this should be fixed in 5.2.0, just released. I'd love to know if people witness the problem go away after upgrading
